traffic_ops/build/traffic_ops.spec

```diff
@@ -33,7 +33,7 @@ URL:              https://github.com/apache/trafficcontrol/
 Vendor:           Apache Software Foundation
 Packager:         daniel_kirkwood at Cable dot Comcast dot com
 AutoReqProv:      no
-Requires:         cpanminus, expat-devel, gcc-c++, golang >= 1.11, libcurl, libpcap-devel, mkisofs, tar
+Requires:         cpanminus, expat-devel, gcc-c++, golang >= 1.14, libcurl, libpcap-devel, mkisofs, tar
```

Comment:
does epel have a golang package for 1.14 yet?

Reply:
It does not. This is just for goose, right? Downgraded this requirement to 1.13 in 14c522e9dc.

Reply:
yeah, I think it is just for goose, which we should be more lenient with since it's an install-time dependency